app/controllers/professors_controller.rb

```diff
@@ -44,7 +46,7 @@ def show
 
     if !@lowest_rated_course.blank? && !@lowest_rated_course.reviews.blank?
       @lowest_rated_course_review =
-        @lowest_rated_course.reviews.order('overall_rating DESC').first
+        @lowest_rated_course.reviews.order('overall_rating').first
```

Comment:
Either line 41 or this line isn't functioning as intended.

If you abstract these lines (removing mentions of highest/lowest) you get this:
```ruby
if !course.blank? && course.reviews.blank?
  course_review = course.reviews.order('overall_rating').first
end
```
If `order('overall_rating')` orders in ascending order, then `@highest_rated_course_review` is taking the lowest-rated course. If it orders in descending order, then `@lowest_rated_course_review` is taking the highest-rated course.